### ISSUE TITLE
texture loading error on server

### ISSUE THREAD

**Issue by opierice** [uItimatech/Pillarger#4]
this mod need server support？i install that in my client and not server, when i join this server, I found that all my textures were incorrect. 
i dont know thats my problem or the server.
![26C1686B431A4784B984DA91AB5D9F82](https://github.com/uItimatech/Pillarger/assets/153950717/9cd61abc-f812-4ed3-873b-96354474e15c)

**Comment by opierice** [comment]
> n

btw, that textures is so awesome in single game

**Comment by uItimatech** [comment]
Yes, the server needs to have the mod too as it modifies the blockstates. However this issue is still concerning. From just the picture it's hard to tell if those blocks are really what they should be. Are the purple blocks really purpur pillars or are those other blocks and the whole texture atlas got messed up ?

**Comment by papamaionise** [comment]
> Yes, the server needs to have the mod too as it modifies the blockstates. However this issue is still concerning. From just the picture it's hard to tell if those blocks are really what they should be. Are the purple blocks really purpur pillars or are those other blocks and the whole texture atlas got messed up ?

Can you make it so mod disables itself in multiplayer if its not installed on server side that be great

**Comment by uItimatech** [comment]
I will look into that. That would be ideal if possible, but I'm really not sure it is.
